hugo/content/blog/2021-03-20-weblate-translation/index.md

```diff
@@ -0,0 +1,75 @@
+
+## Migration to Weblate
+
+The migration itself has mostly technical aspects to it, and a lot of related work. Deep-diving into the localization setup and aspects of our project also showed some inconsistencies and questions for the future, and potential for cleanup which we applied. The migration is/was tracked in [ticket #4727](https://github.com/mumble-voip/mumble/issues/4727).
+
+Weblate provides integration to GitHub and pull request creation. Some advanced configuration scenarios allow for additional customization. Thanks to the integration we were able to phase out our integration and PR creation bot.
+
+Interestingly enough even before finishing the migration and even with notices on Hosted Weblate to not translate yet we received contributions on there.
+
+We have been using Weblate for several weeks now, and have received numerous contributions. Thank you to all existing and future translators!
+
+The process is not as smooth as we would like it to be yet, but we are glad we made the transition already. And we hope existing and future translators like the Hosted Weblate platform.
+
+## Weblate Translation Platform
+
+[Weblate](https://weblate.org/) is Open Source software. The hosted instance services not only us as FOSS but [other projects](https://hosted.weblate.org/projects/) as well.
+
+We are very thankful for the free hosting they provide to us. If you are looking for a translation platform for other or [commercial projects (pricing)](https://weblate.org/en/hosting/), please consider using them. [Direct donations to them](https://weblate.org/en/donate/) are helping them too, and were explicitly mentioned for consideration after approving our project.
+
+## Our Translation Project on Hosted Weblate
+
+We support numerous languages, and also have some languages that are not complete enough for productive use yet. If you are fluent in one of them, please consider contributing to [our translations](https://hosted.weblate.org/projects/mumble/mumble-client/).
+
+![Weblate translation status by language](https://hosted.weblate.org/widgets/mumble/-/mumble-client/horizontal-auto.svg)
+
+If you want to translate a language that we did not add yet, and are willing to commit to it, please create a request in [our issue tracker](https://github.com/mumble-voip/mumble/issues). We can add additional languages.
+
+Have any feedback, positive or negative, praise or concerns? We would love to hear from you!
+
+## Road to Mumble version 1.4
+
+We recently landed blockers for 1.4, and are now in the process of preparing a last development snapshot, a feature freeze, and then plan to publish release candidates.
+
+We would love to release 1.4.0 with many, complete, and good translations. If you are a willing translator, please take this opportunity to focus on finalizing translations towards the upcoming release. It would be great if we had finalized translations within or in a month. Thank you very much.

```

Comment:
```suggestion
We would love to release 1.4.0 with many, complete, and good translations. If you are a willing translator, please take this opportunity to focus on finalizing translations towards the upcoming release. It would be great if we had finalized translations within about a month or two. Thank you very much.

```